code/modules/cargo/packs.dm

```diff
@@ -147,11 +147,27 @@
 	name = "NULL_ENTRY"
 	hidden = TRUE
 	cost = 14000
-	contains = list(/obj/item/weapon/storage/box/syndicate)
+	contains = null
```

Comment:
Iirc the cargo uses empty contains to differentiate between abstract and normal crates. So you need to actually put something in here or change that.